conf/messages.en

```diff
@@ -339,3 +339,18 @@ route.builder.share = Copy link to route
 help.rating.1 = Rating 1: Passable
 help.rating.3 = Rating 3: Neutral
 help.rating.5 = Rating 5: Not passable
+
+home.graphic = A graphic of an intersection with people traveling across the street
+home.graphic.seattle = A skyline graphic of Seattle, Washington
+home.explain = GIF explaining the outcomes of crowdsourcing and machine learning on Project Sidewalk
+
+clouds = clouds
```

Comment:
This should probably say something like "white clouds partially covering the image above, with statistics shown on top of the clouds"